packages/core/src/shared-wallets/add-shared-wallet/SharedWalletLayout/type.ts

```diff
@@ -1,4 +1,4 @@
 export type LayoutNavigationProps = {
-  onBack?: () => void;
-  onNext?: () => void;
+  onBack?: () => Promise<void> | void;
+  onNext?: () => Promise<void> | void;
```

Comment:
We are adding Promise to void because of analytics which I think is unnecessary
```suggestion
  onBack?: () => void;
  onNext?: () => void;
```

Reply:
done here c0fe6a164ccff6290f54af2f906d13031a2d6715